src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/sealedclaim/ResponseRepaymentDetailsForm.java

```diff
@@ -108,6 +110,7 @@ private static void addRepaymentPlan(CaseData caseData, ResponseRepaymentDetails
                                       .build())
                 .payBy(repaymentPlan.finalPaymentBy(totalClaimAmount))
                 .whyNotPayImmediately(caseData.getResponseToClaimAdmitPartWhyNotPayLRspec());
+            builder.amountToPay = (totalClaimAmount + "");
```

Comment:
is there no method for this?

Reply:
I think they are doing it this way to protect against npe

Reply:
Changed to builder.amountToPay(totalClaimAmount + "")